example/spec/parallel/ingestGranule/IngestUMMGSuccessSpec.js

```diff
@@ -216,10 +216,10 @@ describe('The S3 Ingest Granules workflow configured to ingest UMM-G', () => {
       }),
       providerModel.delete(provider),
       executionModel.delete({ arn: workflowExecution.executionArn }),
-      granulesApiTestUtils.removePublishedGranule({
-        prefix: config.stackName,
-        granuleId: inputPayload.granules[0].granuleId
-      })
+      // granulesApiTestUtils.removePublishedGranule({
+      //   prefix: config.stackName,
+      //   granuleId: inputPayload.granules[0].granuleId
+      // })
```

Comment:
Is there a reason this was commented out?

Reply:
Wow, good catch. I was doing some testing on a separate feature and needed to create some granules. Will fix

Reply:
f27cc29036c9f68669d926acc20cbb7292e7227a